compiler/src/org.graalvm.compiler.hotspot.aarch64/src/org/graalvm/compiler/hotspot/aarch64/AArch64HotSpotRegisterAllocationConfig.java

```diff
@@ -101,7 +105,7 @@
         r0,  r1,  r2,  r3,  r4,  r5,  r6,  r7,
         r8,  r9,  r10, r11, r12, r13, r14, r15,
         r16, r17, r18, r19, r20, r21, r22, r23,
-        r24, r25, r26, /* r27, */ r28, /* r29, r30, r31 */
+        r24, r25, r26, r27, r28, r29, r30, r31,
```

Comment:
@pli-arm @gilles-duboscq https://github.com/oracle/graal/issues/1704 is likely due to this change. Thought we have excluded them at `AArch64HotSpotRegisterConfig.initAllocatable`

Reply:
Somehow `r27`, i.e., `heapBaseRegister` is used even when compressed oops is off.